api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -483,16 +483,32 @@ message HttpConnectionManager {
   // Should paths be normalized according to RFC 3986 before any processing of
   // requests by HTTP filters or routing? This affects the upstream *:path* header
   // as well. For paths that fail this check, Envoy will respond with 400 to
-  // paths that are malformed. This defaults to false currently but will default
-  // true in the future. When not specified, this value may be overridden by the
-  // runtime variable
+  // paths that are malformed. This defaults to true.
```

Comment:
v2 is frozen so please make the changes in v3 only.

Reply:
Ack.